Source/FullScreenPokemon.ts

```diff
@@ -3832,23 +3832,44 @@ module FullScreenPokemon {
 
         /**
          * Opens the context menu within the Pokedex menu for the selected Pokemon.
+         *
+         * @param settings   Contains the selected Pokemon.
```

Comment:
Phrasing is off here. Other places are along the lines of `"Settings for ABC, including XYZ"` or "Information on ABC, including XYZ"`.